Tasks/ExtractFiles/extractfilestask.ts

```diff
@@ -200,6 +200,7 @@ function sevenZipExtract(file, destinationFolder) {
     sevenZip.arg('x');
     sevenZip.arg('-o' + destinationFolder);
     sevenZip.arg(file);
+    sevenZip.arg('-aoa'); // Overwrite All existing files without prompt.
```

Comment:
This is a change in behavior that could break existing customers. Were we always overwriting existing files?

Reply:
not overwriting is actually breaking and hanging folks (it prompts, hangs or fails with odd error).

Reply:
The customer is asking us to take that zip and get the contents of it at location xxx.  Not overwriting seems like a really odd inconsistent state as well

Reply:
I understand, just calling out that it is a change in existing behavior and some customers might be relying on previous behavior. Tough call!

Reply:
yeah, we also considered making it an option in inputs.  But if we did all that for compat, then the default would have to be false since it's not obvious when it hangs/fails (I think this actually said something about not enough memory - bad error handling in 7z).  One other option is to add it as an input option but default it to true.  That way if it does affect someone adversely, at least there's a way back (change to false).  OK?

Reply:
I like that, add option but default it to true which will work for most customers based on what we know.

Reply:
We discussed and decided to add an environment variable that a potential affected user could set to unblock.  We believe there won't be any of these issues but technically it could affect someone.  That would unblock.  If we're incorrect and there's a scenario we're not thinking of we can unblock and then immediately follow up with adding an input. Optimistically, we won't ever need the extra input.

Reply:
Spoke with Bryan and Eric and we are going to add a feature flag to turn this off. That allows users to prevent the overwrite if need be without making the change to overbearing by adding an input. @madhurig does that work for you too?

Reply:
@stephenmichaelf: 👍